loops.js

```diff
@@ -51,3 +67,12 @@
 // >> 9
 // ==========================================
 
+for (l = 0; l < 10; l++) {
```

Comment:
De variable wordt niet gedefinieed met ' let '
Zou de variable van een for loop bij i laten beginnen dan j en dan k

Reply:
Hier ga ik op letten! Dan wordt het een stuk duidelijker, dank je wel!